data/mods/CRT_EXPANSION/deadspace/deadspaceitems.json

```diff
@@ -15,7 +15,15 @@
     "coverage": 0,
     "encumbrance": 0,
     "warmth": 0,
-    "artifact_data": { "charge_type": "ARTC_NULL", "effects_worn": [ "AEP_ALL_UP" ] },
+    "//": "Legacy artifact data is set for all stats +2",
+    "relic_data": {
+      "passive_effects": [
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "STRENGTH", "add": 6 } ] },
```

Comment:
your comment says +2 but this is +6?

Reply:
Super glad I made the notes while I was copy pasting values.  Also really happy to be doing this since it's helping me learn how the new relic system works. :)